CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.0.12 (Unreleased)
```

Comment:
```suggestion
## 0.0.13 (Upcoming)

## 0.0.12 (Frebruary 11, 2021)
```

Why not this way?

Reply:
I think the release job will look for upcoming to write the release date and get the correct tag for the next release

Reply:
0.0.12 was not released, so I was planning on just moving the tag

Reply:
Anyways to release the 0.0.13 we need the `Upcoming` one. I think 🤔 
```suggestion
## 0.0.13 (Upcoming)

## 0.0.12 (Unreleased)
```

Reply:
That's correct the script will match against "Upcoming". Do we not have any changes? Should we add a note?

Reply:
```suggestion
## 0.0.12 (Upcoming)
```

Reply:
Does this sound good ? I can also just go for 0.0.13

Reply:
Yeah, you're right, maybe let's not change the release tag